treeherder/perf/management/commands/synthesize_backfill_report.py

```diff
@@ -42,15 +42,13 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         frameworks, repositories, since, days_to_lookup = self._parse_args(**options)
         self._validate_args(frameworks, repositories)
-
         alerts_picker = AlertsPicker(max_alerts=5,
                                      max_improvements=2,
                                      platforms_of_interest=('windows10', 'windows7', 'linux', 'osx', 'android'))
         backfill_context_fetcher = IdentifyAlertRetriggerables(max_data_points=5,
                                                                time_interval=days_to_lookup)
-        reporter = BackfillReportMaintainer(alerts_picker, backfill_context_fetcher)
-
-        reporter.provide_updated_reports(since, frameworks, repositories)
+        perf_sheriff_bot = PerfSfheriffBot(alerts_picker, backfill_context_fetcher)
```

Comment:
We should only provide `BackfillReportMaintainer` as dependency to `PerfSheriffBot`.
It shouldn't be `PerfSheriffBot`'s responsibility of building the maintainer.